### ISSUE TITLE
sources.list empty

### ISSUE THREAD

**Issue by meyskens** [multiarch/ubuntu-debootstrap#1]
When I run the image I notice the sources.list file is empty

```
root@4796ef1a03ec:/# cd /etc/apt/
root@4796ef1a03ec:/etc/apt# ls
apt.conf.d  preferences.d  sources.list  sources.list.d  trusted.gpg  trusted.gpg.d  trusted.gpg~
root@4796ef1a03ec:/etc/apt# cat sources.list
```

**Comment by moul** [comment]
Yes, for now the images were shipped without sources.list

I added the sources.list in the image directly depending on it, for example here: https://github.com/scaleway/image-ubuntu/blob/master/15.10/overlay-armv7l/etc/apt/sources.list

You can create a PR that adds default sources.list files, it may be definitely useful for people willing to start a project from these images

**Comment by moul** [comment]
PS: I suggest you to generate those files dynamically from the update.sh script

**Comment by meyskens** [comment]
Solved with #2
